plugins/woocommerce/src/Blocks/QueryFilters.php

```diff
@@ -32,11 +32,7 @@ public function main_query_filter( $args, $wp_query ) {
 			return $args;
 		}
 
-		if ( $wp_query->get( 'filter_stock_status' ) ) {
-			$args = $this->stock_filter_clauses( $args, $wp_query );
-		}
-
-		return $args;
+		return $this->add_query_clauses( $args, $wp_query );
```

Comment:
We can't do this. The reason we only modify the query clauses for stock is that the WC Query handles price, attribute, and rating already.

Reply:
Reverted this change.